examples/notebooks/beam-ml/data_preprocessing/huggingface_text_embeddings.ipynb

```diff
@@ -0,0 +1,456 @@
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "artifact_location_minilm_with_inference_args = tempfile.mkdtemp(prefix='huggingface_')\n",
+        "\n",
+        "embedding_transform = SentenceTransformerEmbeddings(\n",
+        "        model_name=text_embedding_model_name, columns=['x'],\n",
+        "        inference_args={'convert_to_numpy': False}\n",
+        "        )\n",
+        "\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "  data_pcoll = (\n",
+        "          pipeline\n",
+        "          | \"CreateData\" >> beam.Create(content))\n",
+        "  transformed_pcoll = (\n",
+        "      data_pcoll\n",
+        "      | \"MLTransform\" >> MLTransform(write_artifact_location=artifact_location_minilm_with_inference_args).with_transform(embedding_transform))\n",
+        "\n",
+        "  # The outputs are in the Pytorch tensor type.\n",
+        "  transformed_pcoll | 'LogOutput' >> beam.Map(lambda x: print(type(x['x'])))\n",
+        "\n",
+        "  transformed_pcoll | \"PrintEmbeddingShape\" >> beam.Map(lambda x: print(f\"Embedding shape: {len(x['x'])}\"))\n"
+      ],
+      "metadata": {
+        "id": "xyezKuzY_uLD",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "d09a07d5-55dc-4544-ea75-39b8105a3e5b"
+      },
+      "execution_count": 27,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n",
+            "Embedding shape: 384\n",
+            "<class 'torch.Tensor'>\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Use the model `sentence-transformers/sentence-t5-large` to generate text embeddings. The model uses only the encoder from a `T5-large model`. The weights are stored in FP16. For more information about the model, see [Sentence-T5: Scalable Sentence Encoders from Pre-trained Text-to-Text Models](https://arxiv.org/abs/2108.08877)."
```

Comment:
```suggestion
        "Next, we will use the model `sentence-transformers/sentence-t5-large` to generate text embeddings. The model uses only the encoder from a `T5-large model`. The weights are stored in FP16. For more information about the model, see [Sentence-T5: Scalable Sentence Encoders from Pre-trained Text-to-Text Models](https://arxiv.org/abs/2108.08877)."
```

Is this example showing something meaningfully different from the one above it? If not, I'd cut it. If yes, then I'd emphasize the difference in this paragraph.

Reply:
no, it just uses a bigger model. The above snipper uses a smaller model. I think I could remove the snippet that uses a smaller model.